Cargo.toml

```diff
@@ -4,4 +4,5 @@ members = [
     "math",
     "crypto",
     "proving-system/stark",
+    "gpu",
```

Comment:
Is it going to be shared between multiple GPUs ? Else I'd call this metal. Or metal GPU. Or it can be like proving-system, where you have a folder with multiple workspaces

Reply:
Discussed in #190